### PR TITLE
Add Cluster picker

### PR DESCRIPTION
**Pull request by JasperYH**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/cluster-picker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+package:
+  name: cluster-picker
+  version: 1.2.3
+
+build:
+  number: 0
+
+source:
+  fn:  ClusterPicker_1.2.3.jar
+  url: http://hiv.bio.ed.ac.uk/clusterPicker/ClusterPicker_1.2.3.jar
```
Comment (bgruening) [inline recipes/cluster-picker/meta.yaml]: can you add a checksum?

File: recipes/cluster-picker/run_test.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+echo -e ">seq1\nATTA\n>seq2\nATTA\n>seq3\nATTA\n>seq4\nATTA" > dummy.fasta
```
Comment (bgruening) [inline recipes/cluster-picker/run_test.sh]: You could add these 3 lines also to the commands-test section into meta.yaml. Cleanup is not needed.

File: recipes/cluster-picker/cluster-picker.sh [hunk 3]
```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```
Comment (bgruening) [inline recipes/cluster-picker/cluster-picker.sh]: You might want to look at: https://bioconda.github.io/guidelines.html#java

There is a reference to a similar script in python that is maybe more readable and maintainable.

**Comment by JasperYH** [comment]
Thanks, I'll update in another pull request shortly.

**Pull request merged**